.github/workflows/build-on-windows.yml

```diff
@@ -0,0 +1,286 @@
+      VULKAN_SDK: ${{ needs.install-vulkan-sdk-and-runtime.outputs.VULKAN_SDK }}
+
+    defaults:
+      run:
+        shell: cmd
+
+    steps:
+      - name: 🤘 Checkout Code
+        uses: actions/checkout@v2 # https://github.com/actions/checkout
+
+      #- name: 🔽 Get latest CMake and Ninja
+      #  uses: lukka/get-cmake@latest
+
+      - name: ℹ Show Tool Versions
+        run: |
+          date /T
+          time /T
+          curl -V
+          cmake --version
+          msbuild /version
+          vcpkg version
+          echo NUMBER_OF_PROCESSORS=%NUMBER_OF_PROCESSORS%
+
+      - name: 🎯 Cache VULKAN SDK & Runtime
+        id: cache-vulkan
+        uses: actions/cache@v1 # https://github.com/actions/cache
+        with:
+          path: ${{ env.VULKAN_SDK }}
+          key: use-wrong-cache-key-to-trigger-partial-matching-of-restore-keys
+          restore-keys: cache-windows-vulkan
+
+      - name: 🔽 Install Visual Studio Components for C++, CMake, Win10SDK
+        shell: pwsh
+        run: |
+          $VSConfig = Resolve-Path ".\.github\.vsconfig" | select -ExpandProperty Path
+          cd "C:\Program Files (x86)\Microsoft Visual Studio\Installer\"
+          $InstallPath = "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise"
+          $Arguments = ('/c', "vs_installer.exe", 'modify', '--installPath', "$InstallPath", '--config', "$VSConfig", '--quiet', '--norestart', '--nocache')
+          $process = Start-Process -FilePath cmd.exe -ArgumentList $Arguments -Wait -PassThru -WindowStyle Hidden
+          $process.ExitCode
+
+      - name: 📥 Setup VC Environment (➔ vcvarsall)
+        run: call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" %PLATFORM%
+
+      - name: 🔽 Update VCPKG
+        run: |
+          cd C:\vcpkg
+          git pull --quiet
+          bootstrap-vcpkg.bat -disableMetrics
+          vcpkg integrate install --triplet=${{ matrix.config.VCPKG_TARGET_TRIPLET }} --feature-flags=manifests
+          vcpkg version
+
+      - name: 🎯 Cache VCPKG
+        id: cache-vcpkg
+        uses: actions/cache@v1 # https://github.com/actions/cache
+        with:
+          #path: ..\build\vcpkg_installed
+          path: ~\AppData\Local\vcpkg
+          key: cache-windows-vcpkg-${{ matrix.config.VCPKG_TARGET_TRIPLET }}-${{ github.ref }}-${{ github.run_number }}
+          restore-keys: |
+            cache-windows-vcpkg-${{ matrix.config.VCPKG_TARGET_TRIPLET }}-${{ github.ref }}
+            cache-windows-vcpkg-${{ matrix.config.VCPKG_TARGET_TRIPLET }}
+
+      - name: ❔ CHECK important folders, to see if everything is present (before building)
+        run: |
+          dir %VULKAN_SDK%\Bin
+          dir %VULKAN_SDK%\runtime\%PLATFORM%
+          dir %LocalAppData%\vcpkg
+
+      - name: ✏️ CMake ➔ Configure (including VCPKG ➔ Install Dependencies)
+        run: |
+          cmake -G "Visual Studio 16 2019" -A ${{ env.PLATFORM }}                           ^
```

Comment:
As with appveyor it seems artifacts must be relative to the source directory.
It is probably best (which is also what most install manuals would say when building with cmake) to create
a build directory inside the source directory. cd into the build and then do the same without the -B option and with .. as the last argument.

That way you can directly execute build and install from that same directory as well.